backend/app/endpoints.py

```diff
@@ -123,12 +123,28 @@ async def get_portfolio(address: str, start=str):
     )
 
 
-@router.get("/backtest/spread/{address}/{start}/{token_to_divest_from}/{percentage}")
+@router.get(
+    "/backtest/spread/{address}/{start}/{token_to_divest_from}/{spread_token}/{percentage}"
```

Comment:
Let's make this PR mergeable by itself, without the need to wait for the entirety of the frontend changes. Indeed, we're happy when PRs are small and don't wait too long to be merged.

For that, you'd just need to make small change in the frontend code, in file `Product.tsx` line 53:

```typescript
        .slice(0, 10)}/${selectedAsset}/USDC/${percentage}`
```